manifests/globals.pp

```diff
@@ -94,7 +94,13 @@
     'FreeBSD' => '93',
     'OpenBSD' => '9.3',
     'Suse' => $::operatingsystem ? {
-      'SLES' => '91',
+      'SLES' => $::operatingsystemrelease ? {
+        '11.3' => '91',
```

Comment:
I took the liberty to use 93 as default, and assign the 11.x distros to 91 through a regexp. Now it covers all the 11.x series (along with the upcoming sle11sp4) and also sle12 and the upcoming sle12sp1